maltd-frontend/src/App.css

```diff
@@ -0,0 +1,38 @@
+.App {
```

Comment:
Are you planning to use scss or sass ? does it integrate after?

Reply:
we will probably use `scss` for this work - it is simple, we just need to change the filename extension to `.scss` and it will be good to go - good point